docker-compose.yaml

```diff
@@ -4,6 +4,8 @@ services:
   pycryptobot:
     image: ghcr.io/whittlem/pycryptobot/pycryptobot:latest
     container_name: pycryptobot
+    environment:
```

Comment:
I'd not add the environment setting here anymore, since -u should do the same thing, making the env var obsolete

Reply:
Reverted back